app/forms/two_factor_options_form.rb

```diff
@@ -77,7 +72,6 @@ def phone_alternative_enabled?
   def phone_valid?
```

Comment:
Was this method used for anything other than the Kantara restriction (the `validates` block removed above)? Can we remove it?